qiskit_experiments/library/randomized_benchmarking/rb_experiment.py

```diff
@@ -299,21 +298,19 @@ def __compose_clifford_seq(
     def __compose_clifford(
         self, left_elem: SequenceElementType, right_elem: SequenceElementType
     ) -> SequenceElementType:
-        if self.num_qubits <= 2:
-            utils = self._cliff_utils
-            return utils.compose_num_with_clifford(
-                left_elem, utils.create_cliff_from_num(right_elem)
-            )
-
+        if self.num_qubits == 1:
+            return compose_1q(left_elem, right_elem)
+        if self.num_qubits == 2:
+            return compose_2q(left_elem, right_elem)
         return left_elem.compose(right_elem)
```

Comment:
Regarding the call to `compose`, this could be a bit misleading because `left_elem` can be `Integral` according to the interface, but cannot be Integral in practice. I think a comment would be helpful here and in other invocations of `compose`.

Reply:
I've noticed we don't need this function any longer, so removed in https://github.com/Qiskit/qiskit-experiments/commit/851cf64de73b1aa3435d49294c4f79272a8fd0c2